api/src/main/java/org/apache/cloudstack/api/response/CapabilitiesResponse.java

```diff
@@ -100,6 +100,10 @@ public class CapabilitiesResponse extends BaseResponse {
     @Param(description = "true if experimental features for Kubernetes cluster such as Docker private registry are enabled, false otherwise")
     private boolean kubernetesClusterExperimentalFeaturesEnabled;
 
+    @SerializedName("customhypervisordisplayname")
+    @Param(description = "Display name for custom hypervisor")
```

Comment:
do we have the tradition of adding since parameter in response ? If yes, can you please add that. Remaining code LGTM.

Reply:
Done, thanks